src/test/app/AccountDelete_test.cpp

```diff
@@ -912,6 +912,274 @@ class AccountDelete_test : public beast::unit_test::suite
         env.close();
     }
 
+    void
+    testDestCreds()
```

Comment:
nit: can we spell this out a bit more? `testDestinationDepositAuthCredentials` or something?

Reply:
fixed